web-registry/src/lib/db/types/json-ld/project-metadata.ts

```diff
@@ -10,13 +10,7 @@ export interface AnchoredProjectMetadataBaseLD {
   'schema:location': GeocodeFeature;
   'regen:projectType': string;
   'regen:projectActivity': CompactedNameUrl;
-  'regen:offsetGenerationMethod': string;
-  // regen:offsetGenerationMethod doesn't get compacted for project metadata because
-  // in COMPACTED_CONTEXT, we specify: 'regen:offsetGenerationMethod': { '@container': '@list' }
-  // while in project metadata, it's a string, not a list
-  // this context is used in credit classes metadata, we should probably update the credit classes metadata
-  // to use offsetGenerationMethods (plural!) since it can be a list
```

Comment:
we decided to update project metadata `offsetGenerationMethod` to be a list instead, see thread: https://github.com/regen-network/regen-registry-standards/issues/35#issuecomment-1419877284